internal/models/quizzes/quizzes.go

```diff
@@ -166,6 +166,176 @@ func GetQuizzes(db *sql.DB) ([]Quiz, error) {
 	return quizzes, nil
 }
 
+func GetUnfinishedQuizzesByUserID(db *sql.DB, userID uuid.UUID) ([]Quiz, error) {
```

Comment:
Could unfinished quizzes and finished quizzes be the same function with a boolean parameter, and you inject the different query line needed based on that?

Reply:
Feel like it is a bit more clear having two different functions for it, rather than hiding it behind a boolean.

Reply:
Hm, I refactored GetPublishedQuizzes and GetUnpublishedQuizzes into one after Kacper made the suggestion. I think the same applies here. There is a lot of duplicate code and I believe only one line difference.

Reply:
is it possible to conditionally render parts of a query string without removing the protection against query injection? because the only thing that actually changes for now is a < to a =

Reply:
Would something like `if is_finished { sign = '<' } else { sign = '=' }` and then using query parameters like we would for IDs and stuff but with 'sign' work?

Reply:
Reworked it in eede624, is that better?

Reply:
I like that. But let's wait for @nokacper24 to also review it.